tests/kuttl/tests/keystone_scale/01-assert.yaml

```diff
@@ -0,0 +1,244 @@
+#
```

Comment:
If the first step just tests the deployment, this assert file should be just a symlink to `assert_sample_deployment`, to avoid duplication

Reply:
ack, done,  thanks